src/components/ToastContainer.tsx

```diff
@@ -4,7 +4,19 @@ import { Toast } from './Toast';
 import { zIndex } from '../../src/theme';
 import { ToastData } from '../../src/types';
 
-const StyledToastContainer = styled(BodyPortal)`
+const StyledToastContainer = styled.div`
```

Comment:
With this being a regular div again i think you can remove `slot='toast'` in the ToastContainer

Reply:
Good point